plugin.json

```diff
@@ -11,6 +11,22 @@
             "type": "string",
             "default": "",
             "required": true
+        },
+        {
+            "key": "triggeringEvent",
```

Comment:
how about making this a comma-separated list?

Reply:
good idea